Makefile

```diff
@@ -1,37 +1,62 @@
 # Extremely simple HTTP server that responds on port 8000 with a hello message.
 
-DOCKERHUB_ID:=ibmosquito
-NAME:="web-hello-go"
-VERSION:="1.0.0"
+DOCKER_HUB_ID ?= ibmosquito
+SERVICE_NAME:="web-hello-go"
+PATTERN_NAME:="pattern-web-helloworld-go"
+SERVICE_VERSION:="1.0.0"
+ARCH:="amd64"
+
+CONTAINER_CREDS:=
 
 default: build run
 
 build:
-	docker build -t $(DOCKERHUB_ID)/$(NAME):$(VERSION) .
+	docker build -t $(DOCKER_HUB_ID ?= ibmosquito)/$(SERVICE_NAME):$(SERVICE_VERSION) .
```

Comment:
You can remove `?= ibmosquito` everywhere except for line 3. It's only needed at the top to set it initially

Reply:
yess I had done that but not pushed it yet, I would do it asap